curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f4345abe7f2161f99f1ad.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 657f4345abe7f2161f99f1ad
+title: Step 16
+challengeType: 20
+dashedName: step-16
+---
+
+# --description--
+
+Start by redefining the `snake_cased_char_list` variable as an empty list. Make use of the square brace notation to create the list but do not put anything between the braces.
+
+Put the braces in separate lines so that you have some space between them, where you can work on the code for the comprehension.
```

Comment:
```suggestion
Put the braces in separate lines so that you have some space between them, where you can work on the code for the list comprehension.
```